src/PresentationalComponents/Snippets/AdvisoriesIcon.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Flex, FlexItem, Tooltip } from '@patternfly/react-core';
+import propTypes from 'prop-types';
+
+const AdvisoriesIcon = ({ rh, content, Icon }) =>(
+    <Tooltip content={content}>
+        <Flex flex={{ default: 'inlineFlex' }}>
+            <FlexItem spacer={{ default: 'spacerSm' }}>
+                <Icon/>
+            </FlexItem>
+            <FlexItem spacer={{ default: 'spacerSm' }}>
+                {rh.toString()}
+            </FlexItem>
+        </Flex>
+    </Tooltip>
+);
+
+AdvisoriesIcon.propTypes = {
+    Icon: propTypes.element,
+    rh: propTypes.any,
```

Comment:
"rh" does not say much, I think that something like "number" or "count" would be more explanatory. The same for "content", "tooltipText" would be better IMO